dsc_lib/src/configure/mod.rs

```diff
@@ -135,13 +135,13 @@ fn escape_property_values(properties: &Map<String, Value>) -> Result<Option<Map<
     Ok(Some(result))
 }
 
-fn get_progress_bar(len: u64) -> Result<ProgressBar, DscError> {
-    let pb = ProgressBar::new(len);
-    pb.enable_steady_tick(Duration::from_millis(120));
-    pb.set_style(ProgressStyle::with_template(
+fn get_progress_bar_span(len: u64) -> Result<Span, DscError> {
+    let pb_span = warn_span!("");
```

Comment:
Presumably using a warn span means that if tracing level is error, then progress isn't displayed right?  That seems fine but would need to be documented.

Reply:
Correct - added a comment. Wasn't sure where to call that out initially, so it was just in the PR description.